content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -124,6 +124,24 @@ The output shows that `testfile` has group ID 2000, which is the value of `fsGro
 -rw-r--r-- 1 1000 2000 6 Jun  6 20:08 testfile
 ```
 
+Kubernetes by default recursively changes ownership and permission of volume to match `fsGroup` specified in pod's
```

Comment:
Add the feature header that this is an alpha feature in 1.18

Reply:
fixed